Tools/SandboxTest.ps1

```diff
@@ -83,7 +83,35 @@ New-Item $tempFolder -ItemType Directory -ErrorAction SilentlyContinue | Out-Nul
 $WebClient = New-Object System.Net.WebClient
 
 function Get-Release {
-  $releasesAPIResponse = Invoke-RestMethod 'https://api.github.com/repos/microsoft/winget-cli/releases?per_page=100'
+  # TODO: Further improvements
+  # Sending authorized requests with GitHub CLI (aka. `gh.exe`) is much more convenient than using `Invoke-RestMethod`.
+  # It would be better integrate this script with GitHub CLI, since developers do not need creating and configuring PAT manually.
```

Comment:
You can check if gh.exe is available. Examples of how to check if a command is present are in YamlCreate - since it checks for winget.exe in a few instances.

Reply:
Check `gh.exe` with `Get-Command`. If it's available, always prefer using its _GitHub OAuth Token_ (begin with `gho_`) as `$env:GITHUB_TOKEN` for further queries.